images/tier2/change-file.svg

```diff
@@ -1,5 +1,5 @@
 <svg xmlns="http://www.w3.org/2000/svg" height="24" width="24" viewBox="0 0 24 24">
```

Comment:
Yeah fair - I checked with Sarah and they are indeed supposed to be #72777a, but I can do that in a separate one.